CONTRIBUTING.md

```diff
@@ -161,47 +161,47 @@ To minimize unnecessary updates of the local apt database running `apt-get updat
 
 ## Test Architecture
 
-Testing is based on three pilars: A) *Installation of base system*, B) *Test Cases*, and C) *Static analys using linter*.
+Testing is based on three pilars: A) *Installation of base system*, B) *Test Cases*, and C) *Static analysis using linter*.
 
 ### Test installation
-Test installation are done continuously using a Docker on a Travis Virtual Machine and by testing on actual hardware, eg. Raspberry Pi. A docker installation can be performed by three commands. Firstly a docker image is built where the `openhabian` code is injected (see `dockerfile` for details): 
+Test installations are done continuously using Docker on a Travis Virtual Machine and by testing on actual hardware, eg. Raspberry Pi. A Docker installation can be performed by three commands. Firstly a Docker image is built where the `openhabian` code is injected (see `dockerfile` for details):
```

Comment:
I think this part of the documentation is now outdated, since Dockerfile vansished in #683.
@mstormi How shall we proceed here, renaming the x86 back to Dockerfile (which complicates the test) or user Dockerfile.openhabian-x86 here in the docs (commands then need to be changed to use the right Dockerfile)=

Reply:
change docs please to reflect the state after #684

Reply:
opened #856